### PR TITLE
Update "Overview of WebView2 features and APIs" per RelNotes 112

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article sections for review:
* CoreWebView2Settings.**IsReputationCheckingRequired** - added new tab-set in "Security and Privacy" section, which was renamed from "Tracking Prevention":
   * https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2544#security-and-privacy [[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/overview-apis-112/microsoft-edge/webview2/concepts/overview-features-apis.md#security-and-privacy)]
* CoreWebView2PermissionKind.**WindowManagement** - no change, continued just linking to the enum; didn't list this new enum value:
   * https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2544#permissions [[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/overview-apis-112/microsoft-edge/webview2/concepts/overview-features-apis.md#permissions)]

RelNotes 112 Release Stable section:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp#10172245
* CoreWebView2Settings.IsReputationCheckingRequired
* CoreWebView2PermissionKind.WindowManagement

Detailed outline of headings:
```
# Overview of WebView2 features and APIs

## Main classes: Environment, Controller, and Core

## Web/native interop
#### Host/web object sharing
#### Script execution
#### Web messaging
#### Script dialogs
#### Shared buffer
 
## Browser features
#### Printing
#### Cookies
#### Image capture
#### Downloads
#### Permissions
#### Context menus
#### Status bar
#### User Agent
#### Autofill
#### Audio
#### Swipe gesture navigation
#### Fullscreen
#### PDF toolbar
#### Theming
#### Language
#### New window
#### Close window
#### Document title
#### Favicon 
#### Security and privacy  <-- renamed from "Tracking prevention"
 
## Process management
 
## Navigate to pages and manage loaded content
#### Manage content loaded into WebView2
#### Navigation history
#### Block unwanted navigating
#### Navigation events
#### Manage network requests in WebView2
#### Custom scheme registration
#### Client certificates
#### Server certificates
 
## iframes
 
## Authentication
 
## Rendering WebView2 in non-framework apps
#### Sizing, positioning, and visibility
#### Zooming
#### Rasterization scale
#### Focus and tabbing
#### Parent window
#### Keyboard accelerators
#### Default background color
 
## Rendering WebView2 using Composition
#### Connecting to the visual tree
#### Forwarding input
#### Drag and drop
#### Accessibility
 
## User data
#### Multiple profiles
 
## Performance and debugging
 
## Chrome Developer Protocol (CDP)
 
## See also
```

### CODE REVIEW

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 1]
```diff
@@ -550,11 +548,11 @@ See also:
 <!-- ====================================================================== -->
 ###### PermissionKind.WindowManagement
 
-<!-- todo: merge/coordinate this section w/ the above -->
+<!-- 112 todo: merge/coordinate this section w/ the above -->
 
 <!-- this section is from RelNotes 112 -->
 
-The `PermissionKind.WindowManagement` API indicates the kind of a permission request.
+<!-- The `PermissionKind.WindowManagement` API indicates the kind of a permission request.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:555]: probably un-comment in this 112 PR

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ When hosting the WebView2 control, your app has access to the following features
 
 <!-- maintenance notes: add table rows for any new h2 sections -->
 
-<!-- todo: if "Managing SmartScreen" is an h2 section, add a row for it above
+<!-- 112 todo: if "Managing SmartScreen" is an h2 section, add a row for it above
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:35]: probably add this row into above table, in this 112 PR

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 3]
```diff
@@ -550,11 +548,11 @@ See also:
 <!-- ====================================================================== -->
 ###### PermissionKind.WindowManagement
 
-<!-- todo: merge/coordinate this section w/ the above -->
+<!-- 112 todo: merge/coordinate this section w/ the above -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:551]: todo: merge/coordinate this section w/ the above, in this 112 PR.  

@novac42 - probably trivial, probably need to just move the below links into the above tabs and get rid of this h2 section.

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 4]
```diff
@@ -1010,21 +1008,21 @@ See also:
 
 
 <!-- ------------------------------ -->
-#### Controller's script locale
+#### Locale for Javascript
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:1011]: change to:
Locale for JavaScript

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 5]
```diff
@@ -1010,21 +1008,21 @@ See also:
...
 <!-- 
 #### Controller's JavaScript locale
 #### Locale for JavaScript for controller
 #### Controller's default locale
 #### Locale for controller
 
-todo: better wording of heading?  check wording in API Ref docs, is "script locale" a good sequence of words for in sentence?
+Edited heading
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:1018]: after heading settles, delete comment block

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 6]
```diff
@@ -2136,7 +2134,7 @@ Receiver:
 <!-- ====================================================================== -->
 ## Managing SmartScreen
 
-<!-- todo: move this section into the above outline structure.  if this is h2, add a "Managing SmartScreen" row to the nav table at top. -->
+<!-- 112 todo: move this section into the above outline structure.  if this is h2, add a "Managing SmartScreen" row to the nav table at top. -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:2137]: in this 112 PR, move this "Managing SmartScreen" up into the above outline structure, maybe add table row.

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 7]
```diff
@@ -548,6 +548,7 @@ See also:
 <!-- from RelNotes 111: -->
 
 * [CoreWebView2PermissionKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind)
+   <!-- * [CoreWebView2PermissionKind.WindowManagement Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind) -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:551]: @novac42 @peiche-jessica - Should 112 Overview of APIs list the specific enum value CoreWebView2PermissionKind.WindowManagement?

111 Overview didn't list enum value CoreWebView2PermissionKind.MidiSystemExclusiveMessages

Comment (peiche-jessica) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:551]: Just the Enum itself should suffice. If we want to list out enum values we should list out all of them.

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 6 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:2079]: AFAIK [SmartScreen](https://learn.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-overview) is a Windows reputation check product name. ReputationCheck is the API name we decided on during API review. I believe this IsReputationCheckingRequired controls whether SmartScreen is turned on. @nishitha-burman @novac42 please help advise.

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 7 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:551]: All the enum values:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind?view=webview2-dotnet-1.0.1722.45&preserve-view=true#fields
Autoplay
Camera
ClipboardRead
FileReadWrite
Geolocation
LocalFonts
Microphone
MidiSystemExclusiveMessages
MultipleAutomaticDownloads
Notifications
OtherSensors
UnknownPermission
WindowManagement

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: Removing the commented item.  Listing all the above enum members would be as long as the entire tab/section.

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: removed line

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: Renamed "Tracking Prevention" section to "Security and Privacy", added the incoming link there as its own tab-set, deleted "Managing SmartScreen" section.

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: via v111 PR, we ended up only linking to the enum, removed this section / link

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: via v111 PR, deleted section and added its link in Browser > Language section instead.

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: cleaned up via v111 PR

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 6 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]: Renamed "Tracking Prevention" section to "Security and Privacy", added the incoming link there as its own tab-set, deleted "Managing SmartScreen" section.

**Review by peiche-jessica — APPROVED** [review]

**Pull request merged**